content/en/docs/configuration/acme/dns01/digitalocean.md

```diff
@@ -16,7 +16,7 @@ metadata:
   name: digitalocean-dns
 data:
   # insert your DO access token here
-  access-token: "base64 encoded access-token here"
+  access-token: "access-token here"
   ```
```

Comment:
If you prefer to use plain text in the Secret manifest, you can use the `stringData` stanza rather than the `data` stanza,
and then kubectl / kube-apiserver (not sure which) will automatically base64 encode the values for you.

So the documentation as it stands is correct.

https://kubernetes.io/docs/concepts/configuration/secret (search for `stringData`)